my_game.py

```diff
@@ -211,6 +213,14 @@ def game_over(self):
         # Change to game over view
         self.window.show_view(game_over_view)
 
+    def map_pos_change(self, change_x, change_y):
```

Comment:
function map_pos-change should be in player class

Reply:
DONIO!